docs/common/DocsExample.vue

```diff
@@ -123,6 +123,36 @@
         default: false,
         required: false,
       },
+      /**
+       * If true, hides the template (HTML) code block from the code viewer.
```

Comment:
Let's change this to "Flag that hides the template (HTML) code block from the code viewer"